interview-questions.md

```diff
@@ -9,59 +9,60 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is `this` in JavaScript?
 
-  Your answer:
+  Your answer: `This` in JavaScript is a way to refer to an object within itself. It can also be used with dot notation plus a key in order to get a value. 
 
-  Researched answer:
+  Researched answer: It is a keyword in JavaScript. Along with a key, it can create a variable within an object to get a value. 
 
 
 
 2. What is React? Why would you use it?
 
-  Your answer:
+  Your answer: React is a JavaScript library that makes it easy to create many different components and link them all together for a functioning app. It is great to use to allow user interaction because it will only refresh the components for which there is a diff, or a change.
 
-  Researched answer:
+  Researched answer: It is an open source JS library that allows developers to easily create applications with interactive user interfaces. It is component based and it REACTS and makes changes without the need to reload the whole webpage. 
 
 
 
 3. Which lifecycle method is required in a React class component?
 
-  Your answer:
+  Your answer: The render method is required in a React class component. 
 
-  Researched answer:
+  Researched answer: The render method is the only required method in a class component. Its purpose is to display the HTML code to the DOM.
 
 
 
 4. What is JSX? What is one syntax difference between HTML and JSX?
 
-  Your answer:
+  Your answer: JSX is a combination of HTML and JavaScript. One of the differences in syntax between HTML and JSX is that in HTML, you would use a tag with the name of script to run JavaScript in that file. In JSX, you use curly braces {}.
 
-  Researched answer:
+  Researched answer: It is a syntax extension to JavaScript. It is useful because it allows HTML and JS to coexist within the same file with some syntax differences. 
 
 
 
 5. What is yarn? What file(s) are modified in a React application when you run yarn?
 
-  Your answer:
+  Your answer: Yarn is a package manager. It organizes the node modules in a React application. 
 
-  Researched answer:
+  Researched answer: Yarn is used to securely share our code with other developers. When we share our code, it is sent in what is called a package. Yarn manages this package in a secure and fast way. 
 
 
 
 6. STRETCH: What is an anonymous function in JavaScript?
 
-  Your answer:
+  Your answer: An anonymous function is one that does not have a name assigned to it. One of the ways we can see an anonymous function is using arrow syntax. It would look like this:
```

Comment:
Oooo!  Nice job on adding the syntax in this example!  Great work in this section 👏